roles/edpm_iscsid/defaults/main.yml

```diff
@@ -38,4 +38,10 @@ edpm_iscsid_volumes:
   - /etc/target:/etc/target:z
   - /var/lib/iscsi:/var/lib/iscsi:z
 
-edpm_iscsid_chap_algs: 'SHA3-256,SHA256,SHA1,MD5'
+edpm_iscsid_chap_algs: >-
+  {{
+    'SHA3-256,SHA256,SHA1' ~
```

Comment:
Resurfacing my question here about SHA1  -- @Akrog - do we need this, or can we leave it off the list of chap algorithms?

Reply:
SHA1 would be disabled everywhere by default in RHEL9 yeah?
https://www.redhat.com/en/blog/rhel-security-sha-1-package-signatures-distrusted-rhel-9

Not sure why we would need it here. But I'm happy for us to remove it in a follow up PR in the interest of unblocking dependent testing and work.

Reply:
I'll let @Akrog reply to this question.

Reply:
I originally added SHA-1 because I wasn't sure if I could remove it.  That is - I wasn't sure if there were any legacy iSCSI devices that would break if SHA-1 was not offered as an available option.  Thats what I was hoping @Akrog could chime in on.  But to @bshephar  , in that case, we'd likely need to tweak security policies to allow SHA-1 to happen in that case.

I'll have to look into whether this case would be allowed in rhel crypto policies or for FIPS.  I comes down to what the CHAPAlgorithm is being used for.

Reply:
To move things along, we can defer this to a follow-on PR.

Reply:
It's impossible for me to know what every certified vendor hardware and firmware version support.
Evey time we've tried to determine these kind of things based on upstream CI or the RH certification process we've come up with an incorrect answer due to the small sample pool.
Considering we've had troubles in the past with some vendors when disabling MD5, it doesn't sound too far fetched to think that SHA-1 would be the same.

Reply:
Thanks @Akrog and @ASBishop for your replies.

I'll remove MD5 and SHA-1 *only* from FIPS enabled scenario.